migrations/lard/import.go

```diff
@@ -0,0 +1,89 @@
+package lard
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+// TODO: I'm not sure I like the interface solution
```

Comment:
Mmm, do you mean decoupling `LardObs` to separate data and flags timeseries, instead of having them together? That sounds reasonable

Reply:
That's also a good idea, but I meant have methods on the timeseries type that generate [][]any (all the rows at once). Then you could drop the interface and just do `pgx.CopyFromRows(ts.new_method())`. Not sure what the performance implications would be, depends a bit on Go's runtime and pgx's implementation.

Reply:
I did both, now `parseData` returns three `[][]any` slices (for data, text, and flags). The `[][]any` approach should also work without modifications for Kvalobs, so that's cool, thanks!